logging/logger.js

```diff
@@ -1,40 +1,12 @@
 ﻿var winston = require('winston');
```

Comment:
Great thoughts! I will continue to use Winston and follow the patterns found in the boilerplate-api.